sessionctx/variable/session.go

```diff
@@ -379,6 +380,9 @@ type SessionVars struct {
 
 	// LowResolutionTSO is used for reading data with low resolution TSO which is updated once every two seconds.
 	LowResolutionTSO bool
+
+	// Backoff is used for set the back off time.
+	Backoff backoff.Time
```

Comment:
We'd better rename this session variable. In TiDB we have a lot of `backoff`.

Reply:
How about `BackoffVar` to mean this is a variable backoff time?

Reply:
done.